packages/vue/src/index.ts

```diff
@@ -177,10 +177,11 @@ const Frontegg: PluginObject<PluginOptions> | any = {
     }, 10);
 
     const isAuthRoutes = (path: string) => {
+      const pathname = new URL(path, window.location.origin).pathname
       const { routes } = Vue.fronteggAuth;
       return Object.values(routes)
-        .filter(path => path != routes.authenticatedUrl)
-        .includes(path);
+        .filter(route => route != routes.authenticatedUrl)
+        .includes(pathname);
```

Comment:
create generic function and use also for `useFronteggAuthGuard` function
```
isAuthRoutes(path, routes){
 const pathname = new URL(path, window.location.origin).pathname
 routes
   .filter(route => route != routes.authenticatedUrl)
   .includes(pathname);
}
```

Reply:
added to fronteggAuth and replace each with fronteggAuth.isAuthRoutes..